libs/wingsdk/src/redis/redis.ts

```diff
@@ -0,0 +1,49 @@
+import { Construct } from "constructs";
+import { fqnForType } from "../constants";
+import { App, Resource } from "../core";
+
+/**
+ * Global identifier for `Bucket`.
+ */
+export const REDIS_FQN = fqnForType("redis.Redis");
+
+/**
+ * Represents a cloud redis db.
+ *
+ * @inflight `@winglang/sdk.redis.IRedisClient`
+ */
+export abstract class Redis extends Resource {
+  /**
+   * Create a new redis.
+   * @internal
+   */
+  public static _newRedis(scope: Construct, id: string): Redis {
+    return App.of(scope).newAbstract(REDIS_FQN, scope, id);
+  }
+
+  public readonly stateful = false; // TODO: redis persistence
+
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    this.display.title = "Redis";
+    this.display.description = "A Redis server";
+  }
+}
+
+/**
+ * Inflight interface for `Redis`.
+ */
+export interface IRedisClient {
+  /**
+   * Get underlying redis client.
+   * @inflight
+   */
+  ioredis(): Promise<any>;
```

Comment:
@yoav-steinberg , I've notice you chose a different API pattern then the one that is used in our other resource.
We don't do `bucket.get_connection().put(f, s);` we do `bucket.put(f,s)`


consider these two api:
```ts (wing) 
bring cloud;
bring redis;

let redis = new redis.Redis();

// This Proposal 
new cloud.Function(inflight (s:str):str => {
   let connection = redis.ioredis();
   connection.set("wing", "does redis");
   let value = connection.get("wing");
   print("${value}");
}) as "test:1";


// cloud.* style
new cloud.Function(inflight (s:str):str => {
   redis.set("wing", "does redis");
   let value = redis.get("wing");
   print("${value}");
}) as "test:2";
```

Is this because of a technical issue or a deliberate design decision?